### PR TITLE
Added test cases related to the JIRA task WPE-468

### PR DESCRIPTION
**Pull request by sandeep-vedam**
Added below test cases related to WPE-468:
FrameworkSnapshot001, 
FrameworkSnapshot002, 
FrameworkSnapshot003, 
FrameworkSnapshot004

### CODE REVIEW

File: src/framework/Framework_Snapshot_001.test.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  restartFramework,
+  screenshot,
+  webKitBrowserOps,
+} from '../commonMethods/commonFunctions'
+import constants from '../commonMethods/constants'
+
+export default {
+  title: 'Framework snapshot test',
+  description: 'Tests if the Framework snapshot module works',
+  teardown: restartFramework,
```
Comment (wouterlucas) [inline src/framework/Framework_Snapshot_001.test.js]: No need

File: src/framework/Framework_Snapshot_001.test.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    },
+    {
+      description: 'Activating Snapshot Plugin',
+      test: pluginActivate,
+      params: constants.snapshotPlugin,
+      assert: 'activated',
+    },
+    {
+      description: 'Deactivating and Activating WebKitBrowser plugin',
+      test: webKitBrowserOps,
```
Comment (wouterlucas) [inline src/framework/Framework_Snapshot_001.test.js]: Same as others, probably good to rename.

**Review by wouterlucas — CHANGES_REQUESTED** [review]

File: src/framework/Framework_Snapshot_001.test.js (hunk 1 above)
Comment (sandeep-vedam) [reply src/framework/Framework_Snapshot_001.test.js]: Remove this code related to restartFramework

File: src/framework/Framework_Snapshot_001.test.js (hunk 2 above)
Comment (sandeep-vedam) [reply src/framework/Framework_Snapshot_001.test.js]: Rename this to webKitBrowserStartAndResume

**Comment by sandeep-vedam** [comment]
@wouterlucas , I have updated the code as per the review comments. Please review and take appropriate actions if it's OK

**Pull request merged**